osclib/adi_command.py

```diff
@@ -89,19 +89,23 @@ def create_new_adi(self, wanted_requests, by_dp=False):
                     continue
 
                 non_ring_packages.append(target_package)
-                if by_dp is True:
-                    devel_project = self.get_devel_project(source_project, source_package)
-                    # try target pacakge in Factory
-                    # this is a bit against Leap development in case submissions is from Update,
-                    # or any other project than Factory
-                    if devel_project is None and self.api.project.startswith('openSUSE:'):
-                        devel_project = self.get_devel_project('openSUSE:Factory', target_package)
-                    if devel_project is not None:
-                        source_project = devel_project
-
-                if not source_project in non_ring_requests:
-                    non_ring_requests[source_project] = []
-                non_ring_requests[source_project].append(request_id)
+                if split:
+                    # request_id pretended to be a source project
+                    non_ring_requests[request_id] = [request_id]
```

Comment:
no, source_project is a real project path, in ```by_dp``` we needs it to find the devel project if possible, but since --split will ignores --by-develproject, should not have problem in ```by_dp```.

actually the only place are effected by *source_project/request_id* is ```for source_project, requests in non_ring_requests.items():```, but it just using as index, no matter what is the content of ```source_project```.